### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by IamYunusAli**
in these Pull request

used Media queries to 
Adjust element on the mobile version to work on desktop one

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -403,3 +415,135 @@ textarea::placeholder {
   font-size: 15px;
   padding: 10px;
 }
```
Comment (sja-thedude) [inline style.css:434]: - [x] Kindly fix the mobile version as you can see below in the video when I increase the size there is space and the mobile version doesn't adjust correctly. Make sure to make your design responsive as a desktop one. You have to increase the mobile version width to fix it.

https://user-images.githubusercontent.com/70269507/171366653-724035e6-bf20-4096-8fcd-2951ababe035.mov

- [x] Kindly fix the card image in mobile version, as per the Figma design the second card image in your design should be displayed on the 4th card and vice versa. As shown below: 
_Left side is the figma design and right side is your design_

https://user-images.githubusercontent.com/70269507/171368323-af155bf8-5fee-4a29-95a5-dde8569794ed.mov

File: style.css [hunk 2]
```diff
@@ -403,3 +415,135 @@ textarea::placeholder {
...
+    flex-direction: column;
+    width: inherit;
+  }
+
+  .headline {
+    height: 100vh;
+    background-image: url(assets/headerbg.png);
+    background-size: contain;
+    background-repeat: no-repeat;
+    border-bottom-left-radius: 160px;
```
Comment (sja-thedude) [inline style.css]: DESKTOP VERSION -

- [x] Kindly fix the spacing as per shown in videos 1 and 2, following the Figma design. For the intro section and the about section.
- [x] Kindly make sure that your card image's height is as per the Figma design. When your design is increased images are getting smaller, kindly fix it.
- [x] Kindly make sure your card displays an image in 50% of its width and text takes 50% of the card.
- [x] Kindly follow the spacing for the about section and fix the margin for it when the desktop size increases. Your design at 768px is perfect and when the size increases from 768, the elements should display in the same way following the spacing.
- [x] Kindly follow the same card images as per the Figma design.

Video 1:

https://user-images.githubusercontent.com/70269507/171370081-173f8b71-a3c4-4432-acaa-b916cb37a779.mov

Video 2:

https://user-images.githubusercontent.com/70269507/171370032-874d9547-19a9-46d8-9f7e-5edab6f9ab74.mov

File: index.html [hunk 3]
```diff
@@ -163,7 +176,9 @@ <h1>About  <br>Myself</h1>
                             <li class="list-item">
                                 <img src="assets/css.png" alt="">
                                 HTML</li>
-                            <li class="list-item"><img src="assets/html.png" alt="">CSS</li>
+                            <li class="list-item">
+                              <img src="assets/html.png" alt="">
+                              CSS</li>
```
Comment (sja-thedude) [inline index.html]: - [x] Kindly fix the images displayed for HTML and CSS. You can see below, that you have mistakenly added HTML image to CSS and CSS to HTML. Kindly fix it.

https://user-images.githubusercontent.com/70269507/171372916-dab4d523-c63c-4be0-b886-d52e0b573e75.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @IamYunusAli, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ All linters are passing
✔️ Good PR Title and description

## Required 
- [x] Kindly update your readme with the current project description and screenshots.
- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @IamYunusAli , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**